src/components/pages/landing/Gooddollar.js

```diff
@@ -29,96 +30,104 @@ export const Gooddollar = ({ setShowGooddollarVerification }) => {
+        wallet_identifier: wallet.accountId,
+        g$_address: data.gDollarAccount,
+        status: 'Approved',
+      };
+      log_event({
+        event_log: `Data sent to verify API ${JSON.stringify(sendObj)}`,
+      });
+      try {
+        const result = await verifyUser(sendObj);
+        const { data } = await supabase.select('users', {
           wallet_identifier: wallet.accountId,
-          g$_address: data.gDollarAccount,
-          status: 'Approved',
-        };
-
-        try {
-          const result = await verifyUser(sendObj);
-          const { data } = await supabase.select('users', {
-            wallet_identifier: wallet.accountId,
-          });
-
-          if (data?.[0]) {
-            const { error: appError } = await supabase.update(
-              'users',
-              updateData,
-              { wallet_identifier: wallet.accountId }
-            );
-          } else {
-            const { error: appError } = await supabase.insert(
-              'users',
-              updateData
-            );
-          }
-          if (result?.m) {
-            await wallet.callMethod({
-              contractId: gooddollar_contract,
-              method: 'sbt_mint',
-              args: {
-                claim_b64: result.m,
-                claim_sig: result.sig,
-              },
-              deposit: '8000000000000000000000',
-            });
-            log_event({ event_log: 'Applied for OG SBT' });
-          } else {
-            setIsVerifyApiError(true);
-          }
-        } catch (e) {
-          console.log('Error', e);
-          toast.error(
-            'An error occured while submitting your details , please try again'
+        });
+        log_event({
+          event_log: `Data receivied from verify API ${JSON.stringify(result)}`,
+        });
+        if (data?.[0]) {
+          const { error: appError } = await supabase.update(
+            'users',
+            updateData,
+            { wallet_identifier: wallet.accountId }
+          );
+        } else {
+          const { error: appError } = await supabase.insert(
+            'users',
+            updateData
           );
-        } finally {
-          setSubmitting(false);
         }
-      },
-      validate: (values) => {
-        const errors = {};
-        // if (values.phone) {
-        //   if (!isPossiblePhoneNumber(values.phone)) {
-        //     errors.phone =
-        //       "Invalid phone number, please provide the phone number with valid country code";
-        //   }
-        // } else {
-        //   errors.phone = "Phone Number Required";
-        // }
-        return errors;
-      },
-    });
+        log_event({ event_log: 'Applied for FV SBT' });
+        await wallet.callMethod({
+          contractId: gooddollar_contract,
+          method: 'sbt_mint',
+          args: {
+            claim_b64: result.m,
+            claim_sig: result.sig,
+          },
+          deposit: '8000000000000000000000',
```

Comment:
this should be a config. Please create a config file with constants we are using.

Reply:
fixed